src/components/advertisementFeature/article.tsx

```diff
@@ -18,6 +18,7 @@ import Logo from './logo';
 import { withDefault, map } from '@guardian/types/option';
 import { pipe2 } from 'lib';
 import RelatedContent from 'components/shared/relatedContent';
+import FooterCcpa from 'components/shared/footerStyles';
```

Comment:
should the `footerStyles` file be `footer`? as it seems to export more than just styles

Reply:
Sure good idea it includes content as well!